contributors/guide/README.md

```diff
@@ -0,0 +1,212 @@
+**OWNER:**
+
+SIG-ContribX
+
+
+# WARNING
+Hello! This is the starting point for our brand new contributor guide, currently underway as per [-issue#6102](https://github.com/kubernetes/website/issues/6102) and in need of help. Please be patient, or fix a section below that needs improvement, and submit a pull request!
+
+Many of the links below should lead to relevant documents scattered across the community repository. Often, the linked instructions need to be updated or cleaned up. 
+
+* If you do so, please port the relevant file from its previous location to come live in the community/contributors/guide folder, and delete its previous location.
+* Ultimately, all contributor guide specific files will live in this folder.
+
+Please find _Improvements needed_ sections below and help us out.
+
+For example:
+
+_Improvements needed_
+* kubernetes/kubernetes/contributing.md -> point to this guide
+
+* kubernetes/community/CONTRIBUTING.md -> Needs a rewrite 
+
+* kubernetes/community/README.md -> Needs a rewrite 
+
+* Individual SIG contributing documents -> add a link to this guide
+
+* Generate ToC for the section below 
+
+# Welcome
+
+Welcome to Kubernetes! This document is the single source of truth for how to contribute to the code base. Please leave comments / suggestions if you find something is missing or incorrect.
+
+# Before you get started
+
+## Sign the CLA
+
+Before you can contribute, you will need to sign the [Contributor License Agreement](https://github.com/kubernetes/community/blob/master/CLA.md).
```

Comment:
Relative links work, and help when you're working in a fork or branch. So something like `/CLA.md` in this markdown file should link to the right place. You can do a find/replace for anything `https://github.com/kubernetes/community/blob/master/` or `https://github.com/kubernetes/community/tree/master/` and subsitute it with `/`.

Reply:
I'd keep the full path if this document will be moved somewhere.

Reply:
Oh, if the goal is to move this out of community, then I agree.

In that case, I’d suggest https://git.k8s.io/community/CLA.md (and similar). That’s a redirect we own and provides for a clean, consistent anchor for fully qualified URLs

Reply:
Huh. I was unaware of those. Do all kubernetes github files have such a redirect?

And yes, relative files +1, but _after_ we move them into the /guide folder, which in the case of CLA we should.

Reply:
git.k8s.io is just an nginx redirector.. code is here: https://github.com/kubernetes/k8s.io/blob/8320db3841550deb6084137e0533fdc8c193d9b7/k8s.io/configmap-nginx.yaml#L183-L193

and yes, it works for any github.com/kubernetes/ repo